test/.gitignore

```diff
@@ -0,0 +1,3 @@
+package-lock.json
```

Comment:
Everything under this is tests, and not important to the point of es modules support.